tasks/build.js

```diff
@@ -3,7 +3,7 @@ const dutil = require("./utils/doc-util");
 const { buildSprite } = require("./svg-sprite");
 const { compileSass } = require("./sass");
 const { compileJS } = require("./javascript");
-const { copyTheme, copyFonts, copyImages, copySass } = require("./copy");
+const { copyTheme, copyFonts, copyIcons, copyImages, copySass } = require("./copy");
```

Comment:
We could also do something like site where we give tasks a generic name and assign to a single `module.exports`.

Example from [uswds/uswds-site#1611](https://github.com/uswds/uswds-site/pull/1611/)

```js
// config/gulp/javascript.js
function lint(){};
function build(){};
function watch(){};

module.exports = {
  lint,
  build,
  watch
}
```

```js
// gulpfile.js
const js = require("./config/gulp/javascript");

exports.js = gulp.series(js.lint, js.build);
exports.lintJS = js.lint;
exports.watchJS = js.watch;
```